scripts/deploy/deploy.sh

```diff
@@ -26,3 +26,11 @@ echo;
 echo "⚙  ⚙  ⚙  Migrating the database.";
 heroku run rake db:migrate --app "$DEMO_HEROKU_APP_NAME"
 echo;
+
+echo "⚙  ⚙  ⚙  Running Service Rake Task for Demo App.";
```

Comment:
@Jngai Just noticed this change. We don't want to add these lines to `deploy.sh` because  `deploy.sh` is meant to be used for every single deploy. We don't want to run the service migration on every single deploy, just once.

Reply:
@Jngai Sorry for not noticing this issue on my first review!

Reply:
@alexsoble I thought I make it easier for you. I removed them so you have to run the commands manually.